### PR TITLE
Tools: install g++-multilib on Jammy

### PR DESCRIPTION
**Pull request by peterbarker**
Can't build sitl-periph-gps without this

### CODE REVIEW

File: Tools/environment_install/install-prereqs-ubuntu.sh [hunk 1]
```diff
@@ -242,6 +242,10 @@ elif [ ${RELEASE_CODENAME} == 'groovy' ] ||
          [ ${RELEASE_CODENAME} == 'impish' ]; then
     BASE_PKGS+=" python-is-python3"
     SITL_PKGS+=" libpython3-stdlib" # for argparse
+    if [ ${RELEASE_CODENAME} == 'jammy' ]; then
+        # for build.SITLPeriphGPS:
+        BASE_PKGS+=" g++-multilib"
```
Comment (peterbarker) [inline Tools/environment_install/install-prereqs-ubuntu.sh:247]: I wonder if this should be `SITL_PKGS`

**Review by khancyr — COMMENTED** [review]
I don't think this is only for Jammy.
But if we install this, it nukes other cross compiler like gcc-aarch64-linux-gnu gcc-arm-linux-gnueabihf ... And we will need to test this on ARM target.
Even with multilib installed we have an error : 
/usr/include/bits/ioctls.h:23:10: fatal error: asm/ioctls.h: Aucun fichier ou dossier de ce type
do we need dpkg --add-architecture i386 ? to have linux-libc-dev:i386  ?

Considering how low is the demand for sitl_ap_periph, I rather make the error more explicit on waf that try to solve this ...

**Comment by peterbarker** [comment]
> 

Don't we use the release compile for cross-compilation to ARM?

**Comment by peterbarker** [comment]
We've had changes around simulated periph stuff, so this is probably no longer relevant.

**Pull request closed without merging**